fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -206,8 +222,11 @@ pub trait PackFlowValveState {
     fn pack_flow_valve_air_flow(&self, pack_id: usize) -> MassRate;
 }
 
-pub trait GroundSpeed {
-    fn ground_speed(&self) -> Velocity;
+pub trait AdirsSignalInterface {
```

Comment:
I put it in `shared` in case it was useful for other systems in the future 😃, but no I agree at the moment it's only used in the pressurization system. I've renamed it and moved it to `air_conditioning` as suggested.